.gitignore

```diff
@@ -1 +1,4 @@
 credentials.properties
+.idea/
+target/
+csh-pings.iml
```

Comment:
These are specific to your project, please put these in your `.git/info/exclude` file.